calendar/README.md

```diff
@@ -0,0 +1,97 @@
+# Calendar
+
+This folder contains the calendar-API server for NavigaTUM.
+
+This is separated from the server because:
+
+- it has virtually no shared dependencies (natural fault line)
+- this way, we can deploy the calendar-API independently of the main server.
+  The Reason why this is important is, that scraping calendar entries is expensive for TUMOnline.
+  => We have to be resourcefully and can't waste this scraped state by redeploying at will
+  => Making this a StatefulSet instead of a Deployment makes sense
+
+## Getting started
+
+### Prerequisites
+
+For getting started, there are some system dependencys which you will need.
+Please follow the [system dependencys docs](resources/documentation/Dependencys.md) before trying to run this part of our project.
+
+### How to Set up the Sqlite Database (needed for the `get`, `legacy_redirect` and `preview` endpoints)
+
+#### Getting the data
+
+To populate the database, you will need to get said data.
+There are multiple ways to do this, but the easiest way is to download the data from our [website](https://nav.tum.de/).
+
+(Assuming you are in the `server` directory)
+
+```bash
+mkdir -p data
+wget -P data https://nav.tum.de/cdn/api_data.json
+```
+
+Alternatively, you can run the `data` part of this project and generate this file by that part of our docs.
+To link the output directory to the server data directory, so that you don't need to copy on every update you can use:
+
+```bash
+ln -s ../data/output data
+```
+
+#### Setting up the database
+
+To set up the database, you will need to run the `load_api_data_to_db.py` script:
+
+```bash
+python3 load_api_data_to_db.py
+```
```

Comment:
```suggestion
```

I think this is no longer required for this readme

Reply:
Since the clendar needs this database, this is sadly a requirement.
Possibly building an internal endpoint into the main api to provide a list of ids with have calendars attached would be a cleander solution, but this should be another PR

Reply:
You're right, I forgot that it also needs to know the entries where the calendars belong to